### PR TITLE
Added email Verification alert and emailVerification guard

### PR DESCRIPTION
**Pull request by Meesala-vamsi**

### CODE REVIEW

**Comment by Chandandc** [comment]
testing

File: apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx
Comment (Chandandc) [inline apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx:1]: testing with Abhishek

File: apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+export const EmailAlert:React.FC<EmailAlertProps>=({onClose})=>{
+
+  useEffect(()=>{
+    const timer = setTimeout(()=>{
+      onClose();
+    },5000)
+  },[onClose])
+
```
Comment (Chandandc) [inline apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx:14]: testing 02

File: apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx
Comment (Abhishek-karmakar) [reply apps/client/src/pages/dashboard/settings/_dialogs/emailAlert.tsx:1]: responding.. test

**Pull request closed without merging**